src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/docmosis/GenerateTrialReadyFormHandler.java

```diff
@@ -93,6 +94,15 @@ private void buildDocument(CallbackParams callbackParams, CaseData.CaseDataBuild
         var documents = caseData.getTrialReadyDocuments();
         documents.add(element(caseDocument));
         caseDataBuilder.trialReadyDocuments(documents);
+        addDocumentCreatedDate(role, caseDataBuilder);
```

Comment:
Maybe use the feature toggle that was created in:
https://github.com/hmcts/civil-service/pull/5971/files

Reply:
updated